### ISSUE TITLE
Move code to GitLab or Codeberg

### ISSUE THREAD

**Issue by julianfairfax** [breezy-weather/breezy-weather#616]
### Describe your idea

It would be nice if the project's code were moved to GitLab or Codeberg. GitHub is not an open source platform, and being owned by Microsoft makes it more of a danger to open source projects, rather than a help.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] My idea is not related to a question covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document, or a feature described in the [Homepage](https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md) document.
- [X] I have updated the app to **[version 4.6.12 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by TheGamer1002** [comment]
On the one hand, I completely get this idea. However, I've found GitLab hard to use, and not everyone can open merge requests. It feels a lot more "closed in" than GitHub in terms of UI/UX. Just throwing my two cents in.

**Comment by aamirazad** [comment]
I don't think github has done anything to make open source projects to be in any immediate danger. I think if they were to start doing such things, then the project could be moved elsewhere.

**Comment by MillionsToOne** [comment]
Other than feeding your code to their shitty AI?

**Comment by ikanakova** [comment]
> Other than feeding your code to their shitty AI?

Why should it matter for an open source project when the code is free to read for AI even if it's anywhere else?

**Comment by MillionsToOne** [comment]
> > Other than feeding your code to their shitty AI?
> 
> Why should it matter for an open source project when the code is free to read for AI even if it's anywhere else?

The fact that something is open source doesn't mean it's licensed under some sort of GPL license that you can basically copy the entire shit without consequences.

**Comment by ikanakova** [comment]
> Other than feeding your code to their shitty AI?
> 
> Why should it matter for an open source project when the code is free to read for AI even if it's anywhere else?
> 
> And I wouldn't want the first user to check my repo to be an AI bot that basically harvests the whole repo and copies the hard work I've done without even giving a star. That's just generally cocky behavior.

I agree with that, of course. But I think that all public repositories, regardless of license and whether they are on github, gitlab, codeberg, etc. are already mined by AI. Just like all freely available texts on the internet etc.

**Comment by MillionsToOne** [comment]
> >>> Other than feeding your code to their shitty AI?
> >>
> >> Why should it matter for an open source project when the code is free to read for AI even if it's anywhere else?
> > 
> > And I wouldn't want the first user to check my repo to be an AI bot that basically harvests the whole repo and copies the hard work I've done without even giving a star. That's just generally cocky behavior.
> 
> I agree with that, of course. But I think that all public repositories, regardless of license and whether they are on github, gitlab, codeberg, etc. are already mined by AI. Just like all freely available texts on the internet etc.

Yeah, I mean it would be better to have the codebase on a service that doesn't have it's own AI tool, like copilot, that uses user repos to train its AI. Having your code on Github is like promoting this kind of behavior directly.

**Comment by papjul** [comment]
There are currently 231 forks of Breezy Weather on GitHub, so moving main repo to GitLab and deleting it would change nothing to AI mining.

Even if all forks were to be deleted, nothing stops anyone to make a mirror of the Gitlab / Codeberg repo here, or just fork the app here under a new weather app, including all commit history.
